CodeEdit/Features/Editor/Views/EditorView.swift

```diff
@@ -20,24 +23,32 @@ struct EditorView: View {
 
     @EnvironmentObject private var editorManager: EditorManager
 
-    var editorInsetAmount: Double {
-        let tabBarHeight = EditorTabBarView.height + 1
-        let pathBarHeight = showEditorPathBar ? (EditorPathBarView.height + 1) : 0
-        return tabBarHeight + pathBarHeight
-    }
-
     var body: some View {
+        var shouldShowTabBar: Bool {
```

Comment:
Any reason these are inside the view body?

Reply:
Yes, it can be relatively expensive so it only is recomputed when the view is reinitialized.